frontend/sumo-championship/src/components/organisms/ChampionsPerCategoryTable/AgeCategoryItem.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  ChampionsAgeCategory,
+  ChampionsWeightCategory,
+} from '../../../types/Champion';
+import FaIcon from '../../Atoms/FaIcon';
+import WeightCategoryItem from './WeightCategoryItem';
+
+type Props = {
+  ageCategory: ChampionsAgeCategory;
+};
+
+const AgeCategoryItem = ({ ageCategory }: Props) => {
+  return (
+    <div className="championCategoryItem">
+      <div className="header">
+        <FaIcon name="FaUsers" size={16} />
+        <p className="field semiBold">
+          {ageCategory.minAge} - {ageCategory.maxAge}
+        </p>
+        <p className="semiBold">{ageCategory.ageName}</p>
+      </div>
+
+      <div className="body">
+        {ageCategory.weightsAndGender.map(
+          (category: ChampionsWeightCategory) => (
+            <WeightCategoryItem
+              key={ageCategory.ageName + category.gender + category.maxWeight}
```

Comment:
why such a wild key? could not just be index?

Reply:
It can, but I wanted to make it unique

Reply:
ya okey